wrims-core/build.gradle

```diff
@@ -0,0 +1,97 @@
+
+plugins{
+    id 'antlr'
+    id 'library.deps-conventions'
+    id 'library.java-conventions'
+}
+
+sourceSets {
+    src {
+        antlr {
+            srcDir "src/main/antlr/"
+        }
+        java {
+            srcDir "src/main/java/"
+        }
+    }
+}
+
+dependencies {
+    antlr libs.antlr
+
+    implementation libs.hec.monolith
+    implementation libs.hec.nucleus.metadata
+    implementation libs.guava
+    implementation libs.commons.io
+    implementation libs.xstream
+    implementation libs.javatuples
+    implementation libs.java.object.diff
+    implementation libs.kryo
+    implementation libs.jep
+    implementation libs.libtensorflow
+
+    // dependency ont testng exists in main as well as in test
```

Comment:
How do we want to resolve this? Create a ticket to track?

Reply:
I will create a ticket